main.go

```diff
@@ -12,6 +13,28 @@ import (
 	"github.com/gophercloud/utils/openstack/clientconfig"
 )
 
+var helpString = `Prune stale resources from cloud
+
+Usage: prune [OPTION]...
+
+Options:
+  --resource-ttl <ttl>  Minimum age of resources to prune
```

Comment:
The = is required (Prune's argument parsing is very basic!)

```suggestion
  --resource-ttl=<ttl>  Minimum age of resources to prune. ttl is parsed
                         as a Go duration (e.g. "1h", "30m12s")
```